ui/src/widgets/ui-slider/UISlider.vue

```diff
@@ -63,6 +86,27 @@ export default {
             msg.payload = this.value
             this.$store.commit('data/bind', msg)
             this.$socket.emit('widget-change', this.id, this.value)
+        },
+        onDynamicProperties (msg) {
+            const updates = msg.ui_update
+            if (!updates) {
+                return
+            }
+            if (updates.label) {
+                this.dynamic.label = updates.label
+            }
+            if (updates.thumbLabel) {
+                this.dynamic.thumbLabel = updates.thumbLabel
+            }
+            if (updates.min !== undefined) {
+                this.dynamic.min = updates.min
+            }
+            if (updates.max !== undefined) {
+                this.dynamic.max = updates.max
+            }
+            if (updates.step !== undefined) {
+                this.dynamic.step = updates.step
+            }
```

Comment:
```suggestion
            if (typeof updates.min !== 'undefined') {
                this.dynamic.min = updates.min
            }
            if (typeof updates.max !== 'undefined') {
                this.dynamic.max = updates.max
            }
            if (typeof updates.step !== 'undefined') {
                this.dynamic.step = updates.step
            }
```

at minimum, best practice, though for number fields like these, I'd be tempted to ensure the type is `'number'` or to attempt to parse the value to a number (to avoid the inevitable forum posts about sending "12" and it didnt work as expected)